hack/generate-operator-bundle-operatorhub.py

```diff
@@ -87,16 +87,32 @@ def generate_csv_base(version, prev_version, hive_image):
     if not os.path.exists(version_dir):
         os.mkdir(version_dir)
 
+    owned_crds = []
```

Comment:
We just open the CSV earlier and append directly to it but I guess it doesn't matter either way.